src/main/java/com/unstoppabledomains/resolution/contracts/cns/Registry.java

```diff
@@ -0,0 +1,40 @@
+package com.unstoppabledomains.resolution.contracts.cns;
+
+import com.esaulpaugh.headlong.abi.Tuple;
+import com.unstoppabledomains.exceptions.NSExceptionCode;
+import com.unstoppabledomains.exceptions.NSExceptionParams;
+import com.unstoppabledomains.exceptions.NamingServiceException;
+import com.unstoppabledomains.resolution.contracts.Contract;
+import java.math.BigInteger;
+
+public class Registry extends Contract {
+
+  public Registry(String url, String address) {
+    super(url, address, "registry", "abi/cns");
+  }
+
+  public String getResolver(BigInteger tokenID) throws Exception {
+    Object[] args = new Object[1];
+    args[0] = tokenID;
+    Tuple answ = this.fetchMethod("resolverOf", args);
```

Comment:
It would be good idea to have `fetchOne` returing a string as most methods only return one value, not a tuple. The code will be more clean this way.